homeassistant/components/anova_cooker/sensor.py

```diff
@@ -0,0 +1,85 @@
+"""Anova sous vide cooker integration."""
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.helpers.entity import Entity
+
+
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Add sensors for passed config_entry in HA."""
+    async_add_devices([AnovaCookerSensor(config_entry.data["device_id"])])
+
+
+class AnovaCookerSensor(Entity):
```

Comment:
Since anova can run for a long time and be offline for long stretches I would implement the RestoreEntity for this one so that the state is kept across HA reboots and it is not completely empty when you reboot HA while the anova is offline.

Reply:
we're talking to the firebase service in the cloud, not the device itself, so we should get an answer every time, even if the answer is that the device is offline.